docs/maintain/maintain-guides-polkadot-opengov.md

```diff
@@ -244,6 +244,24 @@ Governance > Preimages.
 
 ![preimage](../assets/governance/opengov-preimage-submitted.png)
 
+### Submitting a Preimage to Whitelist
+
+Let's take the example where you would like to change the network protocol, like increasing the active validator count. You could [submit a preimage](#submitting-a-preimage) with the call that sets the number of validators to 1000 and submit a referenda proposal to the Root track directly, or if the Polkadot Technical Fellowship is on board with the proposal, the call could be whitelisted and then be submitted to the Whitelist track in Polkadot OpenGov. Below are the steps to follow when submitting a proposal to the Whitelist track. You [submit a preimage](#submitting-a-preimage) with the call that sets the number of validators encapsulated this way - 
+  `whitelist.dispatchWhitelistedCallWhithPreimage(call)` and obtain preimage hash. If the call is submitted directly, it will not be successfully executed by the Whitelist origin.
+
+![preimage-whitelist](../assets/governance/opengov-submit-preimage-whitelist.png)
+
+- The Polkadot fellowship needs to start a fellowship referendum to whitelist the call with `whitelist.whitelistCall(callHash)`
```

Comment:
Is `callHash` here the Preimage Hash of the above `whitelist.dispatchWhitelistedCallWhithPreimage(call)`?

Reply:
No, it is the hash of the `call`.

Reply:
Would it be helpful if explicitly stated that the call hash can be found at the `encoded call hash` in the JS Apps UI Extrinsics page.